modules/coin/errors.go

```diff
@@ -80,3 +81,11 @@ func ErrNoOutputs() errors.TMError {
 func IsNoOutputsErr(err error) bool {
 	return errors.IsSameError(errNoOutputs, err)
 }
+
+func ErrUnknownKey(mod string) errors.TMError {
```

Comment:
You can also do: `return errors.WithMessage(mod, errUnknownKey, unknownRequest)` just using the basecoin.errors package